### PR TITLE
Add additional information about the use the itemType key in the collection object.

### PR DESCRIPTION
**Pull request by pvretano**

### CODE REVIEW

File: collections/openapi/schemas/collectionInfo.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ properties:
   extent:
     $ref: extent.yaml
   itemType:
-    description: indicator about the type of the items in the collection (the default value is 'unknown').
+    description: indicator about the type of the items in the collection if the collection has an accessible /collections/{collectionId}/items endpoint
     type: string
     default: unknown
```
Comment (cportele) [inline collections/openapi/schemas/collectionInfo.yaml:24]: Note that "unknown" would make it impossible to add a dependency from Features to Common in a future revision of Features Part 1. See #107. Maybe we should resolve that issue in the discussion, too.

File: collections/recommendations/collections/REC_rc-md-item-type.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[[rec_collections_rc-md-item-type]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:rec-id}* |*/rec/collections/rc-md-item-type* 
+^|A |If it is possible and reasonable to interact directly with the items of a collection via an accessible `/collection/{collectionId}/items` endpoint then the `itemType` key SHOULD be included in the collection object to indicate the type of the items (e.g. `feature` or `record`).
```
Comment (cportele) [inline collections/recommendations/collections/REC_rc-md-item-type.adoc:5]: I would simply say "If a resource at the path `/collection/{collectionId}/items` is supported by the API, then the `itemType` ..."

Comment (joanma747) [reply collections/recommendations/collections/REC_rc-md-item-type.adoc]: I suggest to replace the "(e.g...)" by: ". OGC APIs will define values for types of items (e.g. OGC API features defines 'feature' and OGC API record will define 'record')"

Comment (joanma747) [reply collections/recommendations/collections/REC_rc-md-item-type.adoc]: /collection/{collectionId}/items should be /collections/{collectionId}/items`

**Comment by joanma747** [comment]
WE will continue working of items and re-discuss if the path items should be there as well as the itemtype. For the moment the text add coherence to the document

**Pull request merged**